config/options.go

```diff
@@ -1292,15 +1304,15 @@ func (o *Options) GetSigningKey() ([]byte, error) {
 // GetAccessLogFields returns the access log fields. If none are set, the default fields are returned.
 func (o *Options) GetAccessLogFields() []log.AccessLogField {
 	if o.AccessLogFields == nil {
-		return log.DefaultAccessLogFields
+		return log.DefaultAccessLogFields()
```

Comment:
Do you have a sense of how much we should care about the memory allocation for the slice of AccessLogField here? I assume this will be called frequently under production load, but I assume we already make lots of other allocations while in the process of streaming access logs from Envoy.

(no action required)

Reply:
yes this would probably not be great. I can change it to use two pre-allocated arrays.